pkg/server/server_test.go

```diff
@@ -790,6 +790,15 @@ func testServerHistoryError(ctx context.Context, s *ImmuServer, t *testing.T) {
 
 }
 
+func testServerInfo(ctx context.Context, s *ImmuServer, t *testing.T) {
+	h, err := s.ServerInfo(ctx, &schema.ServerInfoRequest{})
+	require.NoError(t, err)
+
+	if !h.GetStatus() {
+		t.Fatalf("ServerInfo, expected %v, got %v", true, h.GetStatus())
+	}
```

Comment:
Could be written as:

```suggestion
	require.True(t, h.GetStatus())
```